src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+
+namespace ReviewService.Abstractions;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1625:Element documentation should not be copied and pasted", Justification = "Returns the parameter itself.")]
```

Comment:
You can probably remove that suppress message by using correct documentation.

Reply:
Done.